graphql/data/marathon/deployments/server.js

```diff
@@ -0,0 +1,19 @@
+import DataLoader from 'dataloader';
+
+import config from '../../../config';
+import { fetchWithAuth } from '../../../utils/fetch';
+
+export default class DeploymentsServerConnector {
+  constructor({ authToken }) {
+
+    const getDeployments = fetchWithAuth(authToken, {
+      baseURI: config.endpoints.deployments
+    });
+
+    this.deploymentsLoader = new DataLoader(getDeployments, { batch: false });
+  }
+
+  get() {
+    return this.deploymentsLoader.load('');
```

Comment:
So by providing an empty string as a key we will be returning the same value for each request, right?